{"body": "\nTry/exception is inside the python code. Can you share part of your file that gives you the problem? What version of Honeybee are you using?\n", "attachments": [], "created_by_name": "Mostapha Sadeghipour Roudsari", "created_at": "July 07, 2016 at 04:19AM", "created_by": "mostapha", "parent_id": "reply_2985220_Comment_1559893", "id": "reply_2985220_Comment_1561526"}